main.py

```diff
@@ -13,13 +13,20 @@ def login():
     if data is None:
         return 'Content-Type is not json.', 400
         
-    if 'name' not in data.keys() or 'password' not in data.keys():
+    if 'name' not in data or 'password' not in data:
         return 'Name or/and password are missing.', 400
     
-    if data['name'] == 'denchik' and data['password'] == 'foobar':
-        return 'Correct name and password.', 200
-    
-    return 'Wrong name or password.', 403
+    credit = load_credits(os.path.dirname(os.path.realpath(__file__)) + '/credentials.txt')
+    if credit == None:
+        return 'Server currently are unavailable.', 500
+
+    if data['name'] not in credit:
```

Comment:
This way the login function depends on the output of `load_credits` (i.e. it will break if the file format changes). Try abstracting the user reading logic & user validation into a separate class.